pyspecProcScripts/phasing.py

```diff
@@ -421,40 +421,3 @@ def hermitian_function_test(
             )
         fl.pop_marker()
     return best_shift
-
-
-def determine_sign(s, direct="t2", fl=None):
-    """Given that the signal resides in `pathway`, determine the sign of the signal.
-    The sign can be used, e.g. so that all data in an inversion-recover or
-    enhancement curve can be aligned together.
-
-    Parameters
-    ==========
-    s: nddata
-        data with a single (dominant) peak, where you want to return the sign
-        of the integral over all the data.
-        This should only contain **a single coherence pathway**.
-    direct: str (default "t2")
-        Name of the direct dimension, along which the sum/integral is taken
-
-    Returns
-    =======
-    data_sgn: nddata
-        A dataset with all +1 or -1 (giving the sign of the original signal).
-        Does *not* include the `direct` dimension
-    """
-    assert s.get_ft_prop(
-        direct
-    ), "this only works on data that has been FT'd along the direct dimension"
-    if fl is not None:
-        fl.push_marker()
-        fl.next("selected pathway")
-        fl.image(s.C.setaxis("vd", "#").set_units("vd", "scan #"))
-    data_sgn = s.C.sum(direct)
-    data_sgn /= zeroth_order_ph(data_sgn)
```

Comment:
I agree with what you're doing, but keep this code, which is more robust (compare this func to the one you are keeping... they are not the same)

Reply:
Actually keep this.  It relies on phasing, so it's not a simple function